j5/components/piezo.py

```diff
@@ -67,7 +67,7 @@ def interface_class() -> Type[PiezoInterface]:
 
     def buzz(self, duration: timedelta, pitch: Pitch) -> None:
         """Queue a note to be played."""
-        if type(pitch) is not int and type(pitch) is not Note:
+        if not (type(pitch) is int or type(pitch) is Note):
```

Comment:
The previous formatting of this is more pythonic.

The bot wants this method split up. I'd recommend creating a static `pitch_to_int` or `validate_pitch` method. Perhaps even both.

Reply:
Ah didn't think of that, thank you!

Reply:
Sorry, I didn't read this properly first time round. I have now done this.